package.json

```diff
@@ -46,7 +46,8 @@
     "iconv-lite": "^0.4.11",
     "readable-stream": "^2.2.6",
     "sprintf": "0.1.5",
-    "sspi-client": "^0.1.0"
+    "sspi-client": "^0.1.0",
+    "node-kerberos": "^0.0.25"
```

Comment:
As mentioned in #624, I don't think having all these different libraries be dependencies of `tedious` is a scalable approach.